src/components/Header.js

```diff
@@ -37,32 +87,55 @@ const Navigation = styled.button`
     cursor: pointer;
     text-decoration: underline;
   }
+
+  @media screen and (max-width: 800px) {
+    padding: 10px;
+  }
 `;
 
-function Header() {
-  return (
-    <Wrapper>
-      <Container>
-        <LogoWraper>
-          <img src={logo} alt="" />
-        </LogoWraper>
-
-        <Navigation>Product</Navigation>
-        <Navigation>Customers</Navigation>
-        <Navigation>Pricng</Navigation>
-        <Navigation>Resources</Navigation>
-
-        <ButtonsWrapper>
-          <Button variant="secondary" size="md">
-            Sign in
-          </Button>
-          <Button variant="primary" size="md">
-            Sign up
-          </Button>
-        </ButtonsWrapper>
-      </Container>
-    </Wrapper>
-  );
+class Header extends React.Component {
+  state = {
+    activeMenu: false,
+  };
+
+  handleClick = () => {
+    const { activeMenu } = this.state;
+    const changeActiveMenu = activeMenu === false ? true : false;
+    this.setState({
+      activeMenu: changeActiveMenu,
+    });
```

Comment:
No need of extra variable, you can do this much simpler:

```suggestion
    this.setState({
      activeMenu: !activeMenu,
    });
```

Reply:
Done